src/olympia/addons/serializers.py

```diff
@@ -242,6 +244,12 @@ def get_categories(self, obj):
     def get_has_eula(self, obj):
         return bool(getattr(obj, 'has_eula', obj.eula))
 
+    def get_is_featured(self, obj):
+        if not hasattr(obj, '_is_featured'):
+            # Any featuring will do.
+            obj._is_featured = obj.is_featured(app=None, lang=None)
```

Comment:
This should mention that this is just a fall back if nothing's set in ElasticSearch, otherwise this is quite confusing (to me)